lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -32,8 +29,9 @@
 }
 
 .FilePickerApp__document-source-buttons {
-  align-items: stretch;
+  @include layout.column($align: 'flex-start');
```

Comment:
This endined being one way to keep the buttons from going full width. I had trouble if the display was set to "flex", I do wonder if there is an even simpler or better approach?